### ISSUE TITLE
Import .wiki files from WikiPad

### ISSUE THREAD

**Issue by Ben136KBC** [giuspen/cherrytree#93]
Please add an Import option to import a series of WikidPad .wiki files. This would mean:

Let the user browse for the directory for the .wiki files are. (This would be the data directory where wikidpad stores the wiki in question.) This directory contains a list of .wiki files, each of which is really a plain text file.

For example, supposed I have a WikidPad wiki, with four topics in my tree:
ChristmasLunch
ChromeInfo
CavettCP
ChangesP4

I would have the four .wiki file attached in the data directory. Each of these can be imported as a node in the cherry tree.

That for starters would be great. I tried importing these files using the plain text files options, but no matter what I did, nothing imported...

[WIkidPadFiles.zip](https://github.com/giuspen/cherrytree/files/101510/WIkidPadFiles.zip)

Pretty please?

**Comment by Ben136KBC** [comment]
I am a software developer, would not mind contributing to cheery tree or implementing this feature myself.

**Comment by giuspen** [comment]
Hi, you can start to clone the git repo and run cherrytree as described in
the home page under "development version".
You need some python skills and many hours. If you want to try start
looking at the modules/imports.py and the class to inherit from zim as an
example.

**Comment by giuspen** [comment]
Is the example you attached an openable wikipad tree?

**Comment by Ben136KBC** [comment]
No, here is a more complete zip file of a sample wiki. This one also uses the gadfly option, which means each page or node is store as a text .wiki file. Note that wikidpad uses sqlite by default, but seems to revert to the gadfly option anyway. Attached is a full zipped up directory.

[MySampleWiki.zip](https://github.com/giuspen/cherrytree/files/102083/MySampleWiki.zip)

I have a large personally wiki set up along the same lines, and it would be ncie to import it into cherrytree. I tried the import text files option in cherrytree, with no luck.

**Comment by giuspen** [comment]
Thank you I'll look at the archive and see if I can make the import work.
I'll let you know

**Comment by 2SI3NX** [comment]
Hi, @giuspen has this worked or have you implemented an import feature from wikidpad? im interested in this as well. Would really like this to work, otherwise it would be a nightmare to re-sort everything.

**Comment by giuspen** [comment]
Hi @2SI3NX, I've been struggling to find free time since my newborn son (2 months old now) but I'll try and give you feedback soon

**Comment by 2SI3NX** [comment]
thanks @giuspen !

**Comment by denis-bogdanas** [comment]
I'm also interested in importing from Wikidpad. I used wikidpad for a few years and have 1000 pages in my wiki. CherryTree looks like the closest thing to wikidpad with active development. I would love to switch and import my data.
